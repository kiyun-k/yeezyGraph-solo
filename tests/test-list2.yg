int main() {
  List<int> i;
  i = new List<int>(1, 2, 3);
  i.printList();
  a = new List<string>("a", "ab", "abc")
  a.printList()
  return 0;
}
