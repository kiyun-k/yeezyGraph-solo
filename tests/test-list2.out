{1, 2, 3}
{a, ab, abc}